### PR TITLE
Display version in ui

### PR DESCRIPTION
**Pull request by fabianschwarzfritz**
Displays the version from package.json in the UI. Where should I put the information in the UI? Any ideas?

Related to #546.

### CODE REVIEW

**Comment by hhaidar** [comment]
@fabianschwarzfritz maybe another dropdown item called `about` or something.

**Comment by funkaoshi** [comment]
You could also display right justified next to the "connected" string.

**Comment by fabianschwarzfritz** [comment]
Okay thanks for the Ideas @hhaidar @funkaoshi :+1: . I'll play around with it and see what looks better. Its the implementation I provided backend wise okay?

**Comment by fabianschwarzfritz** [comment]
The version is now displayed in the status bar, on the right side of connected. I think it's ready to merge :beers: .

File: app/controllers/account.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ module.exports = function() {
     // Routes
     //
     app.get('/', middlewares.requireLogin.redirect, function(req, res) {
+        req.user.version = psjon.version;
```
Comment (hhaidar) [inline app/controllers/account.js]: Can you pass this in directly to the template?

**Comment by hhaidar** [comment]
I like this

**Comment by fabianschwarzfritz** [comment]
With 69e9e37 I am passing the version directly to the _chat.html_ template. Is this what you're talking about?

**Comment by hhaidar** [comment]
@fabianschwarzfritz thank you so much

**Pull request merged**